server/game/cards/09-HoT/SerDontosHollard.js

```diff
@@ -17,8 +17,8 @@ class SerDontosHollard extends DrawCard {
         });
     }
 
-    isControlledLady(card) {
-        return card.location === 'play area' && card.controller === this.controller && card.hasTrait('Lady');
+    isLady(card) {
+        return card.location === 'play area' && card.hasTrait('Lady');
```

Comment:
I have mixed feelings about allowing this to trigger on opponent characters. But you do get a benefit from it (standing Dontos). We should watch for feedback about whether folks find it annoying / confusing.

Reply:
I also suggest not doing this everywhere and freely, but i had a request from the design discord community about this interaction so i figured i could change it.